src/components/ImageInputs.js

```diff
@@ -1,9 +1,10 @@
 import React, { useEffect, useState } from "react";
 import { Block } from "galio-framework";
 import { Text, View } from "react-native";
-import ImagePicker from "./ImagePicker";
+import PickImage from "./ImagePicker";
```

Comment:
Perhaps we should rename `ImagePicker.js` to `PickImage.js` to match the component name change?